test/manual-test-examples/p5.Font/custom/sketch.js

```diff
@@ -1,11 +1,25 @@
+var _webgl = false;
```

Comment:
I'm getting what appear to be errors for this in Chrome. I get console warnings about the number of GL contexts and the dead computer icon inside some of the canvases.

![screenshot 426](https://user-images.githubusercontent.com/10382506/43996534-7f45150c-9d79-11e8-8b91-07744cfc150e.png)

Reply:
yeah, it looks like some browsers limit the number of concurrent webgl contexts. try edge, it doesn't do this.